### DISCUSSION TITLE
Bug: Use multiple cssvars together

### DISCUSSION THREAD

**Discussion by Simon-He95** [tailwindlabs/tailwindcss#13246] [Help]
![image](https://github.com/tailwindlabs/tailwindcss/assets/57086651/90dc3275-5bd0-4b2d-833d-8275849c7b0e)

**Comment by RobinMalfait** [comment]
Hey!

This is intentional, you can use the `scale-*` if you want to scale both the `x` and `y` axis with the exact same value.

If you want to have 2 values for both `x` and `y`, I would recommend to use 2 separate classes:

```html
<div class="scale-x-[var(--scalex)] scale-y-[var(--scaley)]"></div>
```

This way you don't have to guess what each value means in `scale-[foo,bar]`, you also don't have to guess what the separator is. Is it a `,`, an underscore `_`, ...  and so on.

Hope this helps!

**Reply by Simon-He95** [reply]
I know the splitting method, and I hope it can be merged, so that our writing method will be simpler. Maybe it can be used as a new feature. In addition, scale is just one of them. It may be easier to match, such as bg-[#fff,contain,no-repeat,url()], which can match many attributes. Then if there is a better rule to limit
